setup.py

```diff
@@ -396,6 +396,10 @@ def make_relative_rpath(path):
 # Declare extensions and package
 ################################################################################
 
+# nvcc does not support '-fexceptions'
```

Comment:
Actually, it is necessary for that very reason. The idea is to pass `-fexceptions` to all the modules that **does not** call NVCC (since it doesn't support the flag).